### PR TITLE
Namsonx/task/stabi branch

### PR DESCRIPTION
**Pull request by namsonx**

### CODE REVIEW

File: config/robotframework_aio/release_items_RobotFramework_TestsuitesManagement.json [hunk 1]
```diff
@@ -46,14 +46,12 @@
...
-                              "RobotFramework_TestsuitesManagement change 0.9 A",
-                              "RobotFramework_TestsuitesManagement change 0.9 B",
-                              "RobotFramework_TestsuitesManagement change 0.9 X"
+                              "RobotFramework_TestsuitesManagement change 0.9 New RobotFramework 6.1 adaptation; Maintenance of log output"
                             ],
                    "0.10." : [
-                               "RobotFramework_TestsuitesManagement change 0.10 A",
-                               "RobotFramework_TestsuitesManagement change 0.10 B",
-                               "RobotFramework_TestsuitesManagement change 0.10 X"
+                               "RobotFramework_TestsuitesManagement change 0.10 Improve dotdict feature",
```
Comment (HolQue) [inline config/robotframework_aio/release_items_RobotFramework_TestsuitesManagement.json]: Hi Son,

please avoid this massive redundancy: "RobotFramework_TestsuitesManagement change 0.10".

The first version of this file was only a sample I used for testing. But under productive conditions there is no need to put the component name in every row.

File: config/robotframework_aio/release_items_RobotFramework_TestsuitesManagement.json [hunk 2]
```diff
@@ -46,14 +46,12 @@
...
+                               "RobotFramework_TestsuitesManagement change 0.10 Update logging",
+                               "RobotFramework_TestsuitesManagement change 0.10 Code maintenance"
                              ]
```
Comment (HolQue) [inline config/robotframework_aio/release_items_RobotFramework_TestsuitesManagement.json:56]: Hi Son,

"improve", "update", "maintenance". OK, a lot of things happened. And should be mentioned. But what exactly is the outcome for users? These terms are so much common. No one will get a concrete idea about the changes. Is there any chance to go a little bit more in detail? What is improved? What exactly has been changed in the logging?

File: config/robotframework_aio/release_items_RobotFramework_TestsuitesManagement.json (hunk 1 above)
Comment (test-fullautomation) [reply config/robotframework_aio/release_items_RobotFramework_TestsuitesManagement.json]: Hi Son,
important is the end-user perspective.
Means:
Incompatibities, new features or stabilization.

Please follow Holgers Mail: "Release Tool: lessions learned".

Please also update jsonprepro release info.

Thank you,
 Thomas

**Review by test-fullautomation — APPROVED** [review]
Hi Son,
looks good to me!
Thank you,
 Thomas

**Pull request merged**